packages/react-celo/src/modals/connect.tsx

```diff
@@ -125,6 +126,16 @@ export const ConnectModal: React.FC<ConnectModalProps> = ({
     [back, connectionCallback]
   );
 
+  useEffect(() => {
```

Comment:
can you refactor these into a shared hook? 

like `useFixedBody(pendingActionCount > 0)`

Reply:
done!